### PR TITLE
Handling global step increment in optimizer python wrapper

### PR DESCRIPTION
**Pull request by abhinavarora**

### CODE REVIEW

File: python/paddle/v2/framework/optimizer.py [hunk 1]
```diff
@@ -108,6 +109,26 @@ def _get_accumulator(self, name, param):
...
+        assert isinstance(block, framework.Block)
+        assert self._global_step is not None
+        # create the increment op
+        increment_op = block.append_op(
+            type="increment",
+            inputs={"X": self._global_step},
+            outputs={"Out": self._global_step},
+            attrs={"step": 1.0})
+
+        return [increment_op]
```
Comment (jacquesqiao) [inline python/paddle/v2/framework/optimizer.py]: why return a list

Comment (abhinavarora) [reply python/paddle/v2/framework/optimizer.py]: Yes you are right. I should change this to an op

Comment (jacquesqiao) [reply python/paddle/v2/framework/optimizer.py]: 🤝

Comment (abhinavarora) [reply python/paddle/v2/framework/optimizer.py]: Fixed in a747012f47d8fc30b4b53de364bfb818ced104dc

**Pull request merged**